CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Release 94 Ellora (2017-09-27)
+------------------------------
+Redshift: update atomic.events to use ZSTD compression (#3435)
+Redshift: added migration script for 0.8.0 to 0.9.0 (#3440)
+Redshift: Update domain_sessionidx column in atomic.events to accept larger values (#1788)
```

Comment:
should be update not Update

Reply:
this commit isn't going to be in #3438, I'll do it myself.